config.toml

```diff
@@ -782,9 +766,9 @@
 
 [Debug]
     [Debug.InterceptorResolver]
```

Comment:
Do we need all these Debug stuff Enabled = true?

Reply:
Honestly, not sure about it, took this config from our sovereign node. Do you think we should switch it to false?